### ISSUE TITLE
Fails to Wrap istioctl install when using the -f flag

### ISSUE THREAD

**Issue by danehans** [tetratelabs/getmesh#79]
`getmesh` fails to recongnize the `-f` or `--filename` flag for `istioctl install`.

```
$ getmesh istioctl install -f config.yaml -y --verify
error executing istioctl: exit status 64, Error: unknown shorthand flag: 'f' in -f
```

```
$ istioctl version
client version: 1.12.0
control plane version: 1.12.0
data plane version: 1.12.0 (1 proxies)

$ getmesh version
getmesh version: 1.1.3
active istioctl: 1.11.3-tetrate-v0
client version: 1.11.3-tetrate-v0
control plane version: 1.12.0
data plane version: 1.12.0 (1 proxies)
```

xref: https://istio.tetratelabs.io/istio-ca-certs-integrations/cert-manager-integration/

**Comment by danehans** [comment]
xref https://github.com/tetratelabs/istio-distro.io/pull/244

**Comment by psbrar99** [comment]
@danehans : thanks for reporting this, it  is addressed in new release of getmesh cli ( 1.1.4). Please update the getmesh cli to newer version.
